code/post_processing/fine_mapping_post_processing.ipynb

```diff
@@ -1877,7 +1877,7 @@
     "          if(length(conditions) > 0) {\n",
     "  \n",
     "            for(condition in conditions) {\n",
-    "                dat_con <- dat_study${'' if multi_gene else [['condition']]}${[['fsusie_summary']] if fsusie else ''}\n",
+    "                if (${\"FALSE\" if multi_gene else \"TRUE\"}) dat_con <- dat_study[[condition]]${[['fsusie_summary']] if fsusie else ''} else dat_con <- dat_study${[['fsusie_summary']] if fsusie else ''}\n",
```

Comment:
please try to change `[['condition']]` to `'[[condition]]'` or `[[condition]]` to make it looks cleaner. 

And if you want to keep this version, if you specify multi_gene, then it would not be fsusie mode, so the judgement in function in else{} can be deleted.

Reply:
Changed [['condition']] to '[[condition]]', added --step1_only in sos command and successfully tested 2 genes. But floating instance issue exists.